src/content/manifest/overview-manifest.mdx

```diff
@@ -0,0 +1,153 @@
+
+There are other sections to help you build your ideal development experience in Okteto, but this document focuses solely on introducing you to the core concepts of the Okteto manifest. You can learn more about the other Okteto manifest options in the [manifest reference](reference/okteto-manifest.mdx).
+
+## Build
+
+This section contains the instructions for Okteto to build the images for each service in your application. All services defined in this section will appear in the Okteto UI as part of a single development environment that represents the whole of your application. Each individual service will appear as its own development container that’s part of the development environment for your application.
+
+Your `build` section might look like this:
+
+```yaml
+build:
+  base:
+    context: .
+  api:
+    context: api
+  frontend:
+    context: frontend
+    dockerfile: Dockerfile
+    target: dev
+    depends_on: base
+    args:
+      SOURCE_IMAGE: ${OKTETO_BUILD_BASE_IMAGE}
+    secrets:
+      npmrc: .npmrc
+```
+
+This configuration defines images to be built for three services: `base`, `api`, and `frontend`. It also defines a `context` for each image, which tells Okteto which folder/subfolder to use for building each container image. For example: `default (.)` tells Okteto to use the current folder for the `base` image, the `api` subfolder for the `api` image, and the `frontend` subfolder for the `frontend` image.
+
+You can learn more about these additional parameters [here](reference/okteto-manifest.mdx).
+
+## Deploy
+
+This section tells Okteto how, specifically, to deploy your development environment. It typically uses a combination of `helm`, `kubectl`, and `okteto` commands. You can execute multiple commands and label them.
+
+> Note: If you have external resources you'd like to configure and show within the Okteto development environment, you must configure those in this section.
+
+Your `deploy` section might look like this:
+
+```yaml
+deploy:
+    - name: Deploy PostgreSQL
+    command: helm upgrade --install postgresql postgresql/postgresql-11.6.21.tgz -f postgresql/values.yml --version 11.6.21
+    - name: Deploy Frontend
+    command: helm upgrade --install frontend frontend/chart --set image=${OKTETO_BUILD_FRONTEND_IMAGE}
+    - name: Deploy API
+    command: helm upgrade --install api api/chart --set image=${OKTETO_BUILD_API_IMAGE} --set load=${API_LOAD_DATA:-true}
+```
+
+In this section the manifest instructs Okteto to execute three commands: deploy a postgresql database, deploy the `frontend` image (using an environment variable), and deploy the `api` image (also using environment variables).
+
+These commands are executed within the development environment created by Okteto enabling you to create a production-like environment for developing your application.
```

Comment:
Should we add a link to the `deploy` reference docs?

Reply:
```suggestion
These commands are executed within the development environment created by Okteto enabling you to create a production-like environment for developing your application.

You can learn more about this section [here](reference/okteto-manifest.mdx#deploy-string-optional).
```